rules-tests/CodeQuality/Rector/ClassMethod/ReturnTypeFromStrictScalarReturnExprRector/Fixture/return_concat_all_int.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\ClassMethod\ReturnTypeFromStrictScalarReturnExprRector\Fixture;
+
+final class ReturnConcatAllInt
+{
+    public function run()
+    {
+        return 1 . 1;
```

Comment:
this could also be `float 1.1` (meaning the `.` is not a concat op, but a decimal separator

Reply:
1.1 is float, but 1  .  1 is concat; 

ref https://3v4l.org/pFIUa

Reply:
Oh wow. didn‘t realize whitespace is significant in this case. Makes sense

Reply:
Seems suprising enough, that we should have a separate unit case for it?

Reply:
I think not needed, 1.1 is exactly Dnumber :), 1  .  1 is Concat